src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/model/LimitedPartnershipSubmissionDtoValidationTest.java

```diff
@@ -41,6 +43,29 @@ void testCreatePartnershipShouldNotReturnError() {
         assertTrue(violations.isEmpty());
     }
 
+    @Test
+    void testCreatePartnershipShouldReturnBadRequestErrorIfJurisdictionIsInvalid() {
```

Comment:
As with test description above, seems verbose.

Reply:
Will leave this one as is. Verbose, but it makes it clear what's being tested.

Reply:
What about "testCreateWhenJurisdictionIsInvalidReturnsBadRequest"

Reply:
Ok. Have changed the name. See what you think.